### ISSUE TITLE
Quiet Honeycomb reporting writing to STDOUT

### ISSUE THREAD

**Issue by straleyb** [OregonDigital/OD2#696]
### Descriptive summary

Currently, Honeycomb is writing a lot of information to STDOUT. 

```
"{\"meta.beeline_version\"=\u003e\"1.0.0\", \"meta.local_hostname\"=\u003e\"58258bcce84f\", \"service_name\"=\u003e\"od2\", \"name\"=\u003e\"sql.active_record\", \"sql.active_record.sql\"=\u003e\"SELECT  1 AS one FROM \\\"roles\\\" WHERE \\\"roles\\\".\\\"name\\\" = $1 AND (\\\"roles\\\".\\\"id\\\" != $2) LIMIT $3\", \"sql.active_record.name\"=\u003e\"Role Exists\", \"sql.active_record.binds\"=\u003e\"[#\u003cActiveRecord::Relation::QueryAttribute:0x0000561bea6964f8 @name=\\\"name\\\", @value_before_type_cast=\\\"admin\\\", @type=#\u003cActiveModel::Type::String:0x0000561be95a20e8 @precision=nil, @scale=nil, @limit=nil\u003e, @original_attribute=nil, @value=\\\"admin\\\", @value_for_database=\\\"admin\\\"\u003e, #\u003cActiveRecord::Relation::QueryAttribute:0x0000561bea695738 @name=\\\"id\\\", @value_before_type_cast=1, @type=#\u003cActiveModel::Type::Integer:0x0000561be95b26c8 @precision=nil, @scale=nil, @limit=nil, @range=-2147483648...2147483648\u003e, @original_attribute=nil, @value=1, @value_for_database=1\u003e, #\u003cActiveRecord::Attribute::WithCastValue:0x0000561bea6946a8 @name=\\\"LIMIT\\\", @value_before_type_cast=1, @type=#\u003cActiveModel::Type::Value:0x0000561be959de58 @precision=nil, @scale=nil, @limit=nil\u003e, @original_attribute=nil, @value=1\u003e]\", \"sql.active_record.type_casted_binds\"=\u003e\"[\\\"admin\\\", 1, 1]\", \"sql.active_record.statement_name\"=\u003e\"\", \"sql.active_record.connection_id\"=\u003e\"47338940380820\", \"duration_ms\"=\u003e1.3817379949614406, \"trace.trace_id\"=\u003e\"a37786c3-ba59-495f-ac32-4c4f11c3c770\", \"trace.span_id\"=\u003e\"87367a5e-1d94-4ed4-b1f4-0f4a8a605c7c\", \"meta.span_type\"=\u003e\"root\"}"
```

That info needs to be suppressed from STDOUT. There must be some configuration somewhere which can silence the reporting. 

https://docs.honeycomb.io/getting-data-in/ruby/beeline/ Documentation on beeline. 

This work is also dependent on https://github.com/OregonDigital/OD2/pull/677 Which has a configuration change for honeycomb and will probably dictate the work in this ticket and how it will be done. 

### Expected Behavior

No reporting of honeycomb events to STDOUT

### Current Behavior

Reporting to STDOUT from honeycomb

**Comment by straleyb** [comment]
```
config.notification_events = %w[
    sql.active_record
    render_template.action_view
    render_partial.action_view
    render_collection.action_view
    process_action.action_controller
    send_file.action_controller
    send_data.action_controller
    deliver.action_mailer
  ].freeze
```
Can be removed. @decimalator Does this look right?

**Comment by straleyb** [comment]
Closed by #710
